bin/libs/config.ts

```diff
@@ -35,8 +35,55 @@ const cliParameterConfig:string = function() {
     return (value as string);
 }();
 
+function isApimlEnabled(): boolean {
```

Comment:
who calls `isApimlEnabled` and `enableApiml`?

Reply:
Noone currently, the idea was to perform the changes automated in the zowe.yaml but I think the proposal in https://github.com/zowe/launcher/pull/160 is a better solution for this, I will remove this code.